### PR TITLE
(MWPW-130268): add link markup creation.

### PR DESCRIPTION
**Pull request by VKniaz**
Link markup creation.

Resolves: [MWPW-130268](https://jira.corp.adobe.com/browse/MWPW-130268)

Before: https://main--milo--adobecom.hlx.page/drafts/a_vkniaziev/document1

After: https://mwpw-130268--milo--adobecom.hlx.page/drafts/a_vkniaziev/document1

### CODE REVIEW

**Comment by honstar** [comment]
@VKniaz : can you please add `before` and `after` test urls? Thx

File: test/blocks/ost/mocks/wcs-artifacts-mock.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (yesil) [inline test/blocks/ost/mocks/wcs-artifacts-mock.json:1]: this file should be minimised to contain only the minimal set of data and real ids/codes should be replaced with dumb ones.
All non used properties should be removed.

**Review by yesil — CHANGES_REQUESTED** [review]

**Review by honstar — CHANGES_REQUESTED** [review]

**Review by honstar — APPROVED** [review]

**Comment by honstar** [comment]
Just noticed in your `after` test url that the placeholder replacement for a CTA `Buy now` button is still not properly replaced.

Screenshot 2023-05-04 at 10 25 45

**Comment by VKniaz** [comment]
@honstar Document updated, now before on top.

**Comment by yesil** [comment]
@VKniaz  the `CTA-` prefix is still there.
image

**Comment by honstar** [comment]
> @VKniaz the `CTA-` prefix is still there. image

@yesil : this is expected in the `BEFORE` section from @VKniaz's test document where an old CTA link is still in use. The `AFTER` section shows the correct `Buy now` CTA button.

**Comment by yesil** [comment]
@honstar I was checking it at https://mwpw-130268--milo--adobecom.hlx.page/drafts/ilyas/merch

I updated the CTA text and it looks good.
Thanks @VKniaz

**Review by yesil — APPROVED** [review]

**Comment by yesil** [comment]
@VKniaz is it too much ask to update the unit tests?
e.g: https://github.com/adobecom/milo/blob/main/test/blocks/merch/mocks/body.html

**Review by npeltier — APPROVED** [review]

**Pull request merged**